impeller/scene/shaders/unlit.frag

```diff
@@ -17,8 +17,13 @@ in vec4 v_color;
 
 out vec4 frag_color;
 
+float consume_unused() {
```

Comment:
This makes the shader inputs technically consumed until we figure out how to handle this in general.